index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```

Comment:
hi my bro, I'm just testing and adding comments, you are doing well bro!

| Header | Header | Header |
|--------|--------|--------|
| Cell | Cell | Cell |